packages/frontend/src/pages/Tile/index.tsx

```diff
@@ -1,10 +1,51 @@
+import { ITableMetadata, ITableRow } from '@plumber/types'
+
 import { useParams } from 'react-router-dom'
-import { Flex, Text } from '@chakra-ui/react'
+import { useQuery } from '@apollo/client'
+import { Center, Flex, Spinner, Text } from '@chakra-ui/react'
 import Container from 'components/Container'
+import { GET_ALL_ROWS } from 'graphql/queries/get-all-rows'
+import { GET_TABLE } from 'graphql/queries/get-table'
+
+import Table from './components/Table'
 
 export default function Tile(): JSX.Element {
   const { tableId } = useParams<{ tableId: string }>()
 
+  const { data: getTableData, loading: getTableLoading } = useQuery<{
+    getTable: ITableMetadata
+  }>(GET_TABLE, {
```

Comment:
nit: we can combine into a single query TBH, e.g.:

```
getTable(tableId: $tableId) {
  id
  name
  columns {
    id
    name
  }
}
getAllRows(tableId: $tableId) {
  rowId
  data
}
```

or even just adding a rows field to table. that said I can see the advantages of modularizing the queries so this is just a nit.